app/src/main/res/values/strings.xml

```diff
@@ -1,56 +1,109 @@
+    <string name="type_the_code">Enter the verification code sent in the email</string>
+    <string name="didnt_receive">Didn\'t receive?</string>
+    <string name="send_again">Send again</string>
+    <string name="password">Password</string>
+    <string name="login_invalid_password">Invalid Password</string>
+    <string name="save_changes">Save Changes</string>
+    <string name="delete_task">Delete Task</string>
+    <string name="duplicate_task">Duplicate Task</string>
     <string name="yes">Yes</string>
-    <string name="delete_task_confirmation">Are you sure you want to delete the task?</string>
+    <string name="delete_task_confirmation">Are you sure you want to delete this task?</string>
     <string name="no">No</string>
     <string name="cancel">Cancel</string>
     <string name="duplicate_task_confirmation">Are you sure you want to duplicate this task?</string>
-    <string name="menu_configuration">Configurações</string>
-    <string name="menu_goal">Metas</string>
-    <string name="menu_notification">Notificações</string>
-    <string name="menu_logout">Sair da conta</string>
-    <string name="task_name_limit">Quantidade de caracteres máximo, 50!</string>
-</resources>
\ No newline at end of file
+    <string name="menu_configuration">Settings</string>
+    <string name="menu_goal">Goals</string>
+    <string name="menu_notification">Notifications</string>
+    <string name="menu_logout">Log out</string>
+    <string name="task_name_limit">Maximum character limit, 50!</string>
+
+    <!-- API RETURNS -->
+    <string name="api_login_unauthorized">Invalid email and/or password</string>
+    <string name="api_unexpected_error">Oops!!! Something went wrong. Please try again later</string>
+    <string name="api_create_account_already_exist_email">This email is already in use</string>
+    <string name="desc_vertical_logo">Vertical Logo</string>
+    <string name="label_repeat_password">Repeat Password</string>
+    <string name="label_verification_code">Code</string>
+    <string name="desc_horizontal_logo">Horizontal Logo</string>
+    <string name="title_create_account">Create Account</string>
+    <string name="title_login">Access Account</string>
+    <string name="title_add_task">Add Task</string>
+    <string name="label_date_picker">Date</string>
+    <string name="label_time_picker">Time</string>
+    <string name="label_priority_dropdown">Priority</string>
+    <string name="urgent_priority">Urgent</string>
+    <string name="high_priority">High</string>
+    <string name="medium_priority">Medium</string>
+    <string name="low_priority">Low</string>
+    <string name="text_tag_bill">Bill</string>
+    <string name="api_tag_bill">bill</string>
+    <string name="label_category_dropdown">Categories</string>
+    <string name="label_tag_dropdown">Tags</string>
+    <string name="label_task_description">Description</string>
+    <string name="text_tag_candidacy">Candidacy</string>
```

Comment:
I believe "Assignment" is a better name for this tag label. You can change it later.